src/qpmodel.jl

```diff
@@ -121,34 +101,29 @@ end
 Return the structure of the constraints Jacobian in sparse coordinate format in place.
 """
 function NLPModels.jac_coord!(qp :: QuadraticModel, x :: AbstractVector, rows :: Vector{<: Integer},
-                              cols :: Vector{<: Integer}, vals :: Vector{<: AbstractFloat}; kwargs...)
+                              cols :: Vector{<: Integer}, vals :: Vector{<: AbstractFloat})
     vals .= findnz(qp.data.A)[3]
 end
 
-function cons(qp::AbstractQuadraticModel, x :: AbstractVector)
-    c = Vector{eltype(x)}(undef, qp.meta.ncon)
-    cons!(qp, x, c)
-end
-
 function NLPModels.cons!(qp :: AbstractQuadraticModel, x :: AbstractVector, c :: AbstractVector)
     mul!(c, qp.data.A, x)
     qp.counters.neval_jprod += 1
     c
 end
 
-function NLPModels.hprod!(qp :: AbstractQuadraticModel, x :: AbstractVector, v :: AbstractVector, Av :: AbstractVector; kwargs...)
```

Comment:
`hprod` actually has a keyword argument that should be used: `obj_weight`. Also, there is an argument `y`, that should appear in the function call, although it is not used.